TextRPG_Team25/Battle.cs

```diff
@@ -40,7 +41,52 @@ private void PlayerAttack(Monster selected)
                 Console.WriteLine($"HP {currentHp}-> HP {selected.HP}");
             }
 
-            
+  
         }
+
+        private void SpawnEnemy() // 몬스터 소환
+        {
+            Random rand = new Random();
+            int spawnNum = rand.Next(1, 4);
+            RanMonsterSpawn(spawnNum);
+        }
+
+        private void RanMonsterSpawn(int spawnNum) // 랜덤 몬스터 소환
+        {
+            Random rand = new Random();
+            for (int i = 0; i < spawnNum; i++)
+            {
+                int num = rand.Next(0, 3);
+                monster.FieldMonster.Add(monster[num]);
+            }
+        }
+
+        public void StartBattle() // 배틀 시작
+        {
+            SpawnEnemy();
+            for(int i = 0; i < monster.FieldMonster.Count; i++)
```

Comment:
몬스터 클래스가 리스트로 바뀔 예정이므로 미리 알려드립니당! 리스트를 순회할 때에는 foreach문을 쓰면 아주아주 깔끔하게 표현할 수 있습니다!
```suggestion
            foreach (var m in fieldMonsters)
```